examples/hpc-c/hello-newlib-hpc/README.md

```diff
@@ -0,0 +1,21 @@
+# Minimal C program
+
+Just a bare metal C program that makes ude of the Newlib C runtime library.
+
+## Quickstart
+
+1. Build and install Newlib on a directory of your choice and paste the include path and library path the `INC_PATH` and `LIB_PATH` of the CMakeLists.txt, accordingly.
+2. Make sure you've got a compatible `riscv64-unknown-elf-gcc` available on $PATH. 
+3. Then `just run`. Use `just clean-build` to reconfigure from scratch (useful for testing & debugging the build process).
+See Justfile for what goes on behind the scenes.
```

Comment:
There should be a link to headsail-newlib here, so users install the correct version.